microprofile/WFLY-13048_upgrade_microprofile-health_2.2.adoc

```diff
@@ -0,0 +1,59 @@
+= Upgrade MicroProfile Health to 2.2
+:author:            Jeff Mesnil
+:email:             jmesnil@redhat.com
+:toc:               left
+:icons:             font
+:keywords:          microprofile,health,observability
+:idprefix:
+:idseparator:       -
+
+== Overview
+
+
+MicroProfile 3.3 provides a minor upgrade for https://github.com/eclipse/microprofile-health/releases/tag/2.2[MicroProfile Health 2.2].
+
+This component upgrade also covers the upgrade of smallrye-health that implements MicroProfile Health 2.2.
+
+== Issue Metadata
+
+=== Issue
+
+* https://issues.jboss.org/browse/WFLY-13048[WFLY-13048]
+
+=== Related Issues
+
+* https://issues.jboss.org/browse/EAP7-1426[EAP7-1426]
+
+=== Dev Contacts
+
+* mailto:{email}[{author}]
+
+=== QE Contacts
+
+* TBD
+
+=== Testing By
+
+[X] QE
+
+== Requirements
+
+* Upgrade MicroProfile Health to 2.2
+* Upgrade smallrye-health to the version that implements MicroProfile Health 2.2
+* Run any MicroProfile Health test in WildFly integration test suite
+* Run the Eclipse MicroProfile Health TCK without failures
+
+=== Non-Requirements
+
+* Support of the `mp.health.disable-default-procedures` config property is handled separately by to https://issues.jboss.org/browse/WFLY-12952[WFLY-12952] that will add such a default server procedure for the readiness check.
+
+== Implementation Plan
+
+== Test Plan
+
+* Run the WildFly integration basic test suite and checks there are no failures related to MicroProfle Health tests.
+* Run the `microprofile-tck/health` module from WildFly integration test suite and checks that there are no failures.
+
+== Community Documentation
+
+* No community documentation is required for this upgrade
```

Comment:
What about specific behavior for the current integration context (Wildfly/EAP)? 
See, for instance, the usage for what defined at eclipse/microprofile-health#225.

Reply:
The use of this annotation literals do not affect our user and are handled internally. There is no need to add them to our community documentation.

Reply:
Ok for the literals. I am wondering whether the availability of a concrete response classes for client side usage would be worth a mention. WDYT?

Reply:
I don't think so. Beside the community documentation describes how users would configure WildFly to use MicroProfile Health.
Documenting the _use_ of MicroProfile Health is out of scope of the WildFly doc

Reply:
Ok, I get your point and agree.